packages/ckeditor5-engine/tests/view/placeholder.js

```diff
@@ -632,12 +633,62 @@ describe( 'placeholder', () => {
 			enablePlaceholder( {
 				view,
 				element: viewRoot,
-				text: 'foo bar baz',
 				isDirectHost: false
 			} );
 			viewRoot.placeholder = 'new placeholder';
 
 			expect( viewRoot.getChild( 0 ).getAttribute( 'data-placeholder' ) ).to.equal( 'new placeholder' );
 		} );
+
+		it( 'should through warning once if "text" is used as argument', () => {
+			const spy = sinon.spy( console, 'warn' );
+			const warning = 'Deprecation Warning: The "text" argument in the "enableProperty" function will be deprecated ' +
+			'in the next version. Please update your code. Refer to the documentation for alternative usage: ' +
+			'https://ckeditor.com/docs/ckeditor5/latest/updating/guides/update-to-39.html#view-element-placeholder';
+
+			setData( view, '<div></div><div>{another div}</div>' );
+
+			enablePlaceholder( {
+				view,
+				element: viewRoot,
+				isDirectHost: false,
+				text: 'foo bar'
+			} );
+
+			enablePlaceholder( {
+				view,
+				element: viewRoot,
+				isDirectHost: false,
+				text: 'foo bar baz'
+			} );
+
+			sinon.assert.calledOnce( spy.withArgs( warning ) );
+		} );
+
+		it( 'should set placeholder using "text" argument', () => {
+			setData( view, '<div></div><div>{another div}</div>' );
+
+			enablePlaceholder( {
+				view,
+				element: viewRoot,
+				text: 'placeholder'
+			} );
+
+			expect( viewRoot.getAttribute( 'data-placeholder' ) ).to.equal( 'placeholder' );
+		} );
+
+		it( 'should use element\'s placeholder as more prior value', () => {
```

Comment:
```suggestion
        it( 'should prefer element\'s placeholder value over text parameter', () => {
```

Reply:
Good test :+1: we need to assert that new style is indeed preferred over deprecated format.